parcelgen-runtime/src/com/yelp/parcelgen/JsonUtil.java

```diff
@@ -241,4 +259,14 @@ public static Bundle toBundle(Map<String, ? extends Parcelable> input) {
         }
         return output;
     }
+
+    public static <T> Map<String, Map<String, T>> fromBundleTwoLevel(Bundle input, Class<T> claz) {
+        input.setClassLoader(claz.getClassLoader());
```

Comment:
Why do we need this?

Reply:
This is necessary because the existing `fromBundle` method doesn't take into account a Map of Maps.